onnx/backend/test/case/node/meansquaredistance.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+def mean_squared_distance(input, target, reduction='mean', w=None):
```

Comment:
Any numerical comparison between this reference implementation and the pytorch (or tensorflow) corresponding one?

Reply:
import torch
import torch.nn as nn
import numpy as np
from numpy.testing import assert_allclose

def mean_squared_distance(input, target, reduction='mean', w=None):
    out = np.square(input - target)
    if w is not None:
        out = np.multiply(out, w)
    if reduction == 'mean':
        out = np.mean(out)
    elif reduction == 'sum':
        out = np.sum(out)
    return out

loss = nn.MSELoss()
input = torch.randn(3, 5)
target = torch.randn(3, 5)
output = loss(input, target)

my_loss = mean_squared_distance(input.numpy(), target.numpy())
assert_allclose(my_loss, output.numpy(), rtol=1e-05)

Reply:
```
import torch
import torch.nn as nn
import numpy as np
from numpy.testing import assert_allclose

def mean_squared_distance(input, target, reduction='mean', w=None):
    out = np.square(input - target)
    if w is not None:
        out = np.multiply(out, w)
    if reduction == 'mean':
        out = np.mean(out)
    elif reduction == 'sum':
        out = np.sum(out)
    return out

loss = nn.MSELoss()
input = torch.randn(3, 5)
target = torch.randn(3, 5)
output = loss(input, target)

my_loss = mean_squared_distance(input.numpy(), target.numpy())
assert_allclose(my_loss, output.numpy(), rtol=1e-05)
```